### PR TITLE
Switched to webpack 3

### PR DESCRIPTION
**Pull request by jamonholmgren**
Upgrades to WebPack 3, removes some unnecessary dependencies.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +0,0 @@
-{
-  "presets": ["es2015", "es2016", "react"]
-}
```
Comment (jamonholmgren) [inline .babelrc]: This doesn't seem to be necessary now with Webpack.

File: package.json [hunk 2]
```diff
@@ -4,25 +4,21 @@
...
-    "babel-core": "^6.24.1",
+    "babel-core": "^6.26.0",
     "babel-loader": "^7.0.0",
-    "babel-plugin-module-resolver": "^2.7.1",
-    "babel-polyfill": "^6.13.0",
-    "babel-preset-env": "^1.5.1",
-    "babel-preset-es2015": "^6.24.0",
-    "babel-preset-es2016": "^6.22.0",
-    "babel-preset-react": "^6.23.0",
+    "babel-preset-env": "^1.6.0",
```
Comment (jamonholmgren) [inline package.json]: This is recommended now (rather than es2015/16).

File: package.json [hunk 3]
```diff
@@ -4,25 +4,21 @@
   "repository": {},
   "license": "MIT",
   "devDependencies": {
-    "babel-core": "^6.24.1",
+    "babel-core": "^6.26.0",
     "babel-loader": "^7.0.0",
-    "babel-plugin-module-resolver": "^2.7.1",
-    "babel-polyfill": "^6.13.0",
```
Comment (ryanlntn) [inline package.json]: Have you tested this on older browsers? Removing polyfill will likely break stuff on IE.

Comment (yulolimum) [reply package.json]: @ryanlntn at this time we only officially support Chrome for editing. Down the road, we should support all latest browsers. Speaking of, there needs to be some sort of messaging that conveys this to the user. I can work on that.

Comment (silasjmatson) [reply package.json]: IE...

Comment (jamonholmgren) [reply package.json]: It should be pretty possible with the new `babel-preset-env` to support whatever we want. It polyfills back as far as you want it to.

**Pull request merged**